frontend/public/components/node.jsx

```diff
@@ -40,9 +40,9 @@ const Header = props => {
   const isOperatorInstalled = !!_.find(props.data, n => containerLinuxUpdateOperator.isOperatorInstalled(n));
   return <ListHeader>
     <ColHead {...props} className="col-xs-4" sortField="metadata.name">Node Name</ColHead>
-    <ColHead {...props} className={isOperatorInstalled ? 'col-xs-2' : 'col-xs-4'} sortFunc="nodeReadiness">Status</ColHead>
-    { isOperatorInstalled && <ColHead {...props} className="col-xs-3" sortFunc="nodeUpdateStatus">OS Update</ColHead> }
-    <ColHead {...props} className={isOperatorInstalled ? 'col-xs-3' : 'col-xs-4'} sortField="status.addresses">Node Addresses</ColHead>
+    <ColHead {...props} className="col-sm-2 col-xs-4" sortFunc="nodeReadiness">Status</ColHead>
+    <ColHead {...props} className="col-sm-3 hidden-xs" sortFunc="nodeUpdateStatus">OS Update</ColHead>
```

Comment:
Other tables consistently drop columns from right to left. Unless there's a good reason not to, I'd probably do that here as well.

Reply:
I was thinking the IP addresses might be more important, though if that's not the case I'll switch it.